frontend/apps/reader/modules/readerfont.lua

```diff
@@ -1,12 +1,14 @@
-local InputContainer = require("ui/widget/container/inputcontainer")
 local CenterContainer = require("ui/widget/container/centercontainer")
-local Notification = require("ui/widget/notification")
-local MultiConfirmBox = require("ui/widget/multiconfirmbox")
-local Menu = require("ui/widget/menu")
 local Device = require("device")
-local Screen = require("device").screen
-local Input = require("device").input
 local Event = require("ui/event")
+local Geom = require("ui/geometry")
+local GestureRange = require("ui/gesturerange")
+local Input = require("device").input
```

Comment:
nitpick, not your fault, but this can be simplified to `local Input = Device.input`